### ISSUE TITLE
Support for vJoy v2.1.8

### ISSUE THREAD

**Issue by FlyingKaenguru** [briankendall/xboxToVJoy#4]
I'm getting `xboxToVJoy failed to start` (see screenshot) when trying to run `xboxToVJoy.exe`.

![image](https://github.com/briankendall/xboxToVJoy/assets/19647660/c4bd9edc-33d4-4c66-b797-4d4b6a7bfbd6)

Is there an easy fix to support vJoy v2.1.8?

**Comment by briankendall** [comment]
It looks like the app just needs to be rebuilt for the new version of vJoy. That's relatively "easy" if you're comfortable with building a Qt application on Windows. I think you would just need to replace the files in the vJoySDK folder, and then maybe fix a few compiler errors that crop up as a result.

I now realize that there are several new releases / forks of vJoy:
https://github.com/njz3/vJoy/
https://github.com/jshafer817/vJoy
https://github.com/shauleiz/vJoy

Do you know which one you're using?

**Comment by FlyingKaenguru** [comment]
First of all, thank you for your fast reply! =) I was a little concerned since the last change was about seven years ago.

To be honest, I was hoping for a user friendly, quick solution - I have no experience in this area of software development.

I stombled across shauleiz's vJoy version when googling for 'vJoy', so I took the latest `vJoySetup.exe` from https://github.com/shauleiz/vJoy/releases/tag/v2.1.8.39.

**Comment by briankendall** [comment]
You might be able to work around the issue by installing vJoy 2.1.6. For what it's worth it still works fine on my Windows 10 system. Maybe give this one a try: https://sourceforge.net/projects/vjoystick/files/Beta%202.x/2.1.6.20-030216/

At some point I'll see about updating the project to use the later vJoy releases.

**Comment by FlyingKaenguru** [comment]
Yeah, I have thought about that too, might be the be the simplest solution for now.
So, not exactly the result I was hoping for, but one I can live with. Thank you for taking the time to look into this! =)

**Comment by DeathWrench** [comment]
The reasoning I'm guessing is because of compatibility with Joystick Gremlin 
https://whitemagic.github.io/JoystickGremlin/download/
`Change vJoy version check to accept any version greater or equal to 2.1.8`
I have also never built a Qt app in windows so uhh please update. :D
